portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -54,14 +62,23 @@
   private static final String PARAM_TOPIC = "topic";
   private static final String PARAM_CONTENT = "content";
   private static final String PARAM_CURSOR = "cursor";
+  /** Toxicity is measured on a scale from 0 to 1.  This should block most directed insults and profanity. */
```

Comment:
```suggestion
  /* Toxicity is measured on a scale from 0 to 1.  This should block most directed insults and profanity. */
```
Normal comments for fields